docs/news/posts/2024-11-01-aiida-shell.md

```diff
@@ -0,0 +1,314 @@
+
+```console
+verdi presto
+```
+
+If there are no error messages, you should now be good to go to follow the rest of the examples.
+
+## The basic concept
+
+All that is needed to run any program with `aiida-shell` is its `launch_shell_job`` function:
+
+```python
+from aiida_shell import launch_shell_job
+results, node = launch_shell_job('date')
+print(results['stdout'].get_content())
+```
+
+In the snippet above, the [`date` program](https://www.man7.org/linux/man-pages/man1/date.1.html) is used as an example, which is a simple Linux utility that prints the current date.
+The program is simply passed as a string to the `launch_shell_job` function.
+Under the hood, `aiida-shell` will find the location of this executable and write the necessary plugins on-the-fly.
+It then runs the code as one would normally do with AiiDA, and the outputs are parsed.
+The output produced by the code that was written to stdout can be retrieved from the results under the `stdout` key.
+
+This is the most basic example and it is rare that a program does not require any inputs.
+Any arguments that the program takes, can easily be specified with the `arguments` argument as follows:
+
+```python
+from aiida_shell import launch_shell_job
+results, node = launch_shell_job(
+    'date',
+    arguments='--iso-8601'
+)
+print(results['stdout'].get_content())
+```
+
+which should print something like ``2022-03-17``.
+There is a lot more functionality that is supported.
+If you want to learn more, the [how-to guides of the online documentation](https://aiida-shell.readthedocs.io/en/latest/howto.html) are an excellent place to get an overview of available features.
+
+## Real life scientific use case
+
+To make the usefulness and power of `aiida-shell` more concrete, let's now go through a real life use case.
+In the following, we will compute the electronic band structure of gallium arsenide using Quantum ESPRESSO.
+[Quantum ESPRESSO](https://www.quantum-espresso.org/) is a free and open-source an integrated suite of Open-Source computer codes for electronic-structure calculations and materials modeling at the nanoscale.
+
+The workflow consists roughly of four calculations:
+
+1. Computing the charge density of the system self-consistently
+2. Computing the electronic structure along high-symmetry k-points
+3. Post-processing the data to extract the electronic band structure
+4. Plot the electronic band structure
+
+Quantum ESPRESSO employs pseudopotentials to describe the effective potential of the system.
+Each calculation requires a pseudopotential for each element that is part of the system.
+For this example, we will use the pseudopotentials provided on the [website of Quantum ESPRESSO itself](https://pseudopotentials.quantum-espresso.org/legacy_tables/ps-library).
+To make them available to the calculations, we download and store them in a `FolderData` node:
+
+```python
+#!/usr/bin/env runaiida
```

Comment:
Or, alternatively to my suggestion of highlighting `verdi run` above, we could also add this shebang.

Reply:
Yeah, in the example of the aiida-shell docs, I have provided the entire file as script and so the shebang guarantees loading the profile. I will leave this as is, and have added a comment above to use `verdi run` or `verdi shell`.